bundle/jsky.app.ot/src/main/java/jsky/app/ot/progadmin/AdminDialog.java

```diff
@@ -44,8 +37,9 @@ ISPProgram show(IDBDatabaseService database, ISPProgram prog)  {
 
         pane.setMessage(ui);
 
-        JDialog dialog = pane.createDialog(owner, title);
+        JDialog dialog = pane.createDialog(null, title);
```

Comment:
If you follow the original code upstream, owner was always `null`, so I removed the parameter and set it explicitly to `null`.

Using `null` basically tells that the box has no parent but as you said `pane.createDialog(title)` may be better in this case